### ISSUE TITLE
[Feature Request] - Default showing help if no timers are defined

### ISSUE THREAD

**Issue by ondrovic** [techygrrrl/timerrr#17]
If not timers are defined it should default to display the help message

**Comment by techygrrrl** [comment]
Thanks for the suggestion! Yeah, it's not particularly helpful right now for new users. 

Ideally I could add a Help component in bubbletea with an option that can open the Add screen.

**Comment by ondrovic** [comment]
That could work

**Comment by techygrrrl** [comment]
Yeah, in another TUI I have, I use a similar layout with Bubbletea's help components to show some functionality. Something like this would be ideal. 

<img width="1135" alt="image" src="https://github.com/user-attachments/assets/bd62ddbb-ca1e-47a5-82ea-7c30005c8012">
